Source/Input/ConfigManager/ConfigManager.cpp

```diff
@@ -0,0 +1,134 @@
+#include "ConfigManager.h"
+
+
+bool LogCategory::IfElementExists(std::string keyValue) {
+
+	if (params.find(keyValue) != params.end()) {
+		return true;
+	}
+
+	return false;
+}
+
+std::string LogCategory::getValue(std::string keyValue) {
+
+
+	if (IfElementExists(keyValue)) {
+
+		return params[keyValue];
+	}
+
+	std::cerr << "Key value not found" << std::endl;
+	system("PAUSE");
+	exit(1);
```

Comment:
that's not a good idea! don't exit anywhere if an error happened, just print a message and try to cleanup. (also don't use exceptions)